### PR TITLE
added sensors

### PR DESCRIPTION
**Pull request by RobertJansen1**
While at it, i decided to add other sensors from absalom-muc as well. i have not verified all values, but took them from absalom-muc as-is

### CODE REVIEW

File: mhi_ac_ctrl.h [hunk 1]
```diff
@@ -273,10 +316,49 @@ class MhiAcCtrl : public climate::Climate,
...
+            else if (value == 13)
+                protection_state_.publish_state("-");
+            else if (value == 14)
+                protection_state_.publish_state("Condensation prevention control");
+            else if (value == 15)
+                protection_state_.publish_state("Current safe control of inverter secondary current");
+            else if (value == 16)
+                protection_state_.publish_state("Stop by compressor rotor lock");
+            else if (value == 17)
+                protection_state_.publish_state("Stop by compressor startup failure");
```
Comment (ginkage) [inline mhi_ac_ctrl.h:359]: Would it be possible to make a static array with all these strings, instead of copy-pasting the same thing 18 times?

**Comment by RobertJansen1** [comment]
I am not really familiar with this form of code, if you have an example somewhere i can certainly rewrite it

File: mhi_ac_ctrl.h (hunk 1 above)
Comment (RobertJansen1) [reply mhi_ac_ctrl.h]: took the lazy response and asked chatGPT :) tested, and works. code is updated

Comment (RobertJansen1) [reply mhi_ac_ctrl.h]: fixed

**Comment by RobertJansen1** [comment]
can this be merged?

**Pull request merged**